ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -592,12 +635,12 @@ BaseVariantGene.propTypes = {
   showInlineDetails: PropTypes.bool,
   compoundHetToggle: PropTypes.func,
   hasRnaTpmData: PropTypes.bool,
-  rnaSeqData: PropTypes.object,
+  individualGeneData: PropTypes.object,
 }
 
 const getRnaSeqProps = (state, ownProps) => ({
   hasRnaTpmData: getFamiliesByGuid(state)[ownProps.variant.familyGuids[0]]?.hasRnaTpmData,
-  rnaSeqData: getRnaSeqOutilerDataByFamilyGene(state)[ownProps.variant.familyGuids[0]],
+  individualGeneData: getIndividualGeneDataByFamilyGene(state)[ownProps.variant.familyGuids[0]] || {},
```

Comment:
Defining a new object outside of a selector that will be passed as a prop causes unnecessary re-renders and is bad practice. Change the component to handle the case where this is missing

Reply:
Okay. I'll move the `|| {}` and add null checking in `showDetails` and `detailsDisplay`.